src/coreclr/gc/gc.cpp

```diff
@@ -3260,6 +3289,39 @@ void gc_heap::fire_pevents()
 #endif //FEATURE_EVENT_TRACE
 }
 
+// This fires the amount of total committed in use, in free and on the decommit list.
+// It's fired on entry and exit of each blocking GC and on entry of each BGC (not firing this on exit of a GC
+// because EE is not suspended then. On entry it's fired after the GCStart event, on exit it's fire before the GCStop event.
+void gc_heap::fire_committed_usage_events()
```

Comment:
>fire_committed_usage_events [](http://example.com/codeflow?start=14&length=27)

should be called `fire_committed_usage_event`? since you are only firing one event